File: beecrowd/Python/iniciante/1016.py
def calcular_tempo(d):
    diferenca_velocidade = 30

    tempo_horas = d / diferenca_velocidade
    tempo_minutos = tempo_horas * 60
    return int(tempo_minutos)

distancia = int(input())
tempo = calcular_tempo(distancia)
print(f"{tempo} minutos")

File: beecrowd/Cpp/iniciante/1016.cpp
/* 
diferenca_velocidade = 30
tempo_horas = d / diferenca_velocidade
tempo_minutos = tempo_horas * 60
*/
#include <iostream>
using namespace std;

int main(){
    double distancia;
    double dif_velocidade = 30;

    cin >> distancia;

    double tempo_horas = distancia / dif_velocidade;
    double tempo_minutos = tempo_horas * 60;

    cout << tempo_minutos << " minutos" << endl;

    return 0;
}